Tier update (6: "Extreme", 7: "Death")

Autorojo opened the pull request:
"Changed the name of tier 6 from Death to Extreme and added tier 7 as Death."
2020/8/14 -- KZTimer 1.97 / KZTimer-API 1.2.2 / GOKZ 2.10.0

fedemahf committed:
Tier update (6: "Extreme", 7: "Death")

Kidev commented:
Thanks for the notification ! I just saw the API update. I will try to be as generic as them (print the tier number when it is out of bounds), and add the new Exteme tier soon.

Syuks committed:
No vote and Warning added

1st option is a No Vote
2nd and 3rd are just warnings

Also disabled pagination.

Syuks committed:
Custom Translations

Added a copy of the original mapchooser's translation files (en & es only) with extra parameters for the warnings.
